server/src/modules/tasks/tasks.service.ts

```diff
@@ -13,11 +13,21 @@ export class TasksService {
     return true;
   }
 
-  updateTask(id: string) {
+  async updateTask(id: string, data: DataTask) {
+    const result = await this.taskModel.updateOne({ id }, {});
     return id;
   }
```

Comment:
Review if an exec is needed to execute these queries
```suggestion
  async updateTask(id: string, data: DataTask) {
    const result = await this.taskModel.updateOne({ id }, {});
    return id;
  }
```